pkg/network/ebpf/c/tracer/tracer.h

```diff
@@ -29,8 +29,10 @@ typedef enum
 typedef struct {
     __u64 sent_bytes;
     __u64 recv_bytes;
+    __u32 sent_packets;
```

Comment:
I assume this was byte packing? new to this but is there a reason to not just put the largest fields first? perhaps because of the presence of `protocol_stack_t`?